File: python_semicolon/assignment/studentscore.py

counter1 = 0
counter2 = 0
score = 0
for count in range(1,16):
	score = int(input("Enter a score for a student \n"))
	if score >= 45:
		print("this student pass my course")
		counter1 = counter1 + 1
	else:
		print("this student failed my course")
		counter2 = counter2 + 1



print("The total number  of student that passed my course is " , counter1)
print("The total number of student that falied my course is " , counter2)
	

File: python_semicolon/assignment/java/StudentScore.java
import java.util.Scanner;
public class StudentScore{

	public static void main(String...args){
		Scanner sc= new Scanner(System.in);
		
		int sum1 = 0;
		int sum2 = 0;
		int score = 0;

		for(int count=1; count<=15; count++){

			System.out.println("Enter a student score");
			score= sc.nextInt();

			if (score>=45){
				
				System.out.println("this student passed my subject");
				sum1++;

		
			}

			
			else{
				
				System.out.println("this student failed my subject");
					sum2++;

			
			}







}

	System.out.println("The total student that passed my subject is " + sum1);
	System.out.println("The total student that falied my subject is " + sum2);

}

}